src/array/without.js

```diff
@@ -0,0 +1,11 @@
+import { pull } from './pull'
+
+/**
+ * This method is an alias of {@link array.pull}.
```

Comment:
If it's an alias you don't write other metas ? Will it work with autocompletion ?